DataFormats/Common/interface/Ref.h

```diff
@@ -117,7 +117,6 @@ Ref: A template for a interproduct reference to a member of a product_.
 #include "boost/type_traits.hpp"
 #include "boost/mpl/has_xxx.hpp"
 #include "boost/utility/enable_if.hpp"
-
```

Comment:
Why no boost change here?

Reply:
sorry - I should not have left this space change in my PR - I am trying to change enable_if but I didn't manage to succeed yet.

Reply:
Do you plan on reverting this or leave it as is?